### ISSUE TITLE
Time Conductor RT input doesn't dismiss

### ISSUE THREAD

**Issue by charlesh88** [nasa/openmct#3942]
Observed on banner/stable. Sporadically, the real-time mode Time Conductor time input popup won't dismiss when the check button is clicked or Return is pressed. 

![image](https://user-images.githubusercontent.com/1056412/121970382-4cc1f700-cd2b-11eb-8cf9-879083dc0154.png)

This error is also thrown in console:

<img width="819" alt="Screen Shot 2021-06-14 at 4 00 20 PM" src="https://user-images.githubusercontent.com/1056412/121970540-a0344500-cd2b-11eb-8745-45bce6ff1e79.png">

Clicking the box or pressing the Return key will apply the inputs properly, but the popup can only be hidden by clicking its "X" button or outside it. Unclear what causes this to happen, but I've encountered it a number of times. When the problem starts, the only recourse is to refresh the browser.

**Comment by akhenry** [comment]
Potentially fixed by https://github.com/nasa/openmct/pull/3948

**Comment by nikhilmandlik** [comment]
should dismiss popup when user press 'esc',
similar to 'enter/return' press submit the form.

**Comment by charlesh88** [comment]
> should dismiss popup when user press 'esc',
> similar to 'enter/return' press submit the form.

Agree, that's a good idea for an enhancement, but isn't an expected behavior right now.

**Comment by charlesh88** [comment]
Verified fixed Testathon 06-25-21

**Comment by kobe1104** [comment]
Verified fixed Testathon 6/25/2021.

**Comment by unlikelyzero** [comment]
@nikhilmandlik to file an enhancement

**Comment by akhenry** [comment]
Verified Fixed

**Comment by shefalijoshi** [comment]
Verified fixed

**Comment by charlesh88** [comment]
Verified fixed Testathon 07-12-21

**Comment by nikhilmandlik** [comment]
When  real-time mode Time Conductor time input popup is open and when you click on local clock input popup is still on screen (it should dismiss)

**Comment by unlikelyzero** [comment]
Leaving open to fix outside of CDA Testathon

**Comment by unlikelyzero** [comment]
Renamed and reopened to describe the problem
